frontend/app/bestrated/page.tsx

```diff
@@ -1,7 +1,7 @@
 export default function BestRated() {
   return (
-    <main>
```

Comment:
Eikö nää vois olla vaan main, niin ei tarviis laittaa mainin sisään div page-wrapperia. Siirtäis ton page-wrapperin tyylin mainiin sass:issa